db/migrate/20191017185340_add_unique_constraint_to_profile_by_account.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueConstraintToProfileByAccount < ActiveRecord::Migration[5.2]
+  def change
+    add_index(:profiles , %i[account_id name benchmark_id], unique: true)
```

Comment:
Profiles aren't even unique by ID in the DB. Maybe we should add that too?

Reply:
Not sure if you mean something different here, but Profile ID is the primary key, so they're not_null and unique by it:
```
compliance=# \d profiles
                                 Table "public.profiles"
        Column         |            Type             |             Modifiers
-----------------------+-----------------------------+------------------------------------
 id                    | uuid                        | not null default gen_random_uuid()
 name                  | character varying           |
 ref_id                | character varying           |
 policy_id             | uuid                        |
 created_at            | timestamp without time zone | not null
 updated_at            | timestamp without time zone | not null
 description           | character varying           |
 account_id            | uuid                        |
 compliance_threshold  | double precision            | default 100.0
 business_objective_id | uuid                        |
Indexes:
    "profiles_pkey" PRIMARY KEY, btree (id)
    "index_profiles_on_account_id" btree (account_id)
    "index_profiles_on_business_objective_id" btree (business_objective_id)
    "index_profiles_on_name" btree (name)
    "index_profiles_on_policy_id" btree (policy_id
```

Reply:
Ok yeah, idk what I was thinking here :laughing: